centreon-open-tickets/www/modules/centreon-open-tickets/providers/Ivanti/IvantiProvider.class.php

```
@@ -0,0 +1,838 @@
+    * handle every query that we need to do
+    *
+    * @param {array} $info required information to reach the glpi api
+    * @param int|null $offset pagination offset
+    *
+    * @return {array} $curlResult the json decoded data gathered from glpi
+    *
+    * throw \Exception 10 if php-curl is not installed
+    * throw \Exception if we can't get a session token
+    * throw \Exception 11 if glpi api fails
+     */
+    protected function curlQuery($info, ?int $offset = null) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$offset'. (undefined)

(UnusedFormalParameter)